client/my-sites/checkout/checkout/style.scss

```diff
@@ -729,7 +729,7 @@
 .credit-card-payment-box__switch-link {
 	color: var( --color-link-dark );
 	font-style: italic;
-	font-weight: 800;
+	font-weight: 700;
```

Comment:
Is there any reason why this one is `700` instead of `600`?

Reply:
Weird, I think I just missed this one!